pantheon-bundle/src/main/java/com/redhat/pantheon/model/module/DocumentStatus.java

```diff
@@ -1,18 +1,27 @@
 package com.redhat.pantheon.model.module;
 
-import com.redhat.pantheon.model.api.Field;
-import com.redhat.pantheon.model.api.WorkspaceChild;
+import java.util.Calendar;
 
 import javax.inject.Named;
 
-/**
- * Models an instance of status acknowledgement sent by integrated systems
- */
-public interface Status extends WorkspaceChild {
-    @Named("pant:status")
-    Field<String> status();
+import org.apache.jackrabbit.JcrConstants;
+
+import com.redhat.pantheon.model.api.Field;
+import com.redhat.pantheon.model.api.WorkspaceChild;
+import com.redhat.pantheon.model.api.annotation.JcrPrimaryType;
+
+@JcrPrimaryType("pant:documentStatus")
+public interface DocumentStatus extends WorkspaceChild {
```

Comment:
One query - since the status represents status of acknowledgement wouldn't AckStatus or something similar be a better name?

Reply:
AP, thank you for the feedback. this is an effort of co working with ben. We took your advice, and refactored some of the names